worker-batch-ingestion-plugin/src/main/java/com/github/cafdataprocessing/worker/ingestion/IngestionBatchWorkerPlugin.java

```diff
@@ -88,6 +92,20 @@ public IngestionBatchWorkerPlugin()
         simpleModule.addDeserializer(DocumentWorkerDocument.class,
                                      new DocumentWorkerDocumentDeserializer(totalSubdocumentLimit.get()));
         mapper.registerModule(simpleModule);
+
+        validationFile = System.getenv("CAF_VALIDATION_FILE");
+        if (StringUtils.isNotEmpty(validationFile)) {
+            try {
+                fieldValidator = new FieldValidator(validationFile);
+            } catch (final ValidationFileAdapterException validationFileAdapterException) {
```

Comment:
Usually you should just use `ex` for exception variables.  They normally have a very small scope.

```suggestion
            } catch (final ValidationFileAdapterException ex) {
```